website/content/docs/getting-started/deploy-and-login.mdx

```diff
@@ -0,0 +1,198 @@
+resources to assist in connecting to your first target. 
+
+![Setup Wizard](/img/setup-wizard-target.png)
+
+The **Manual** setup drops the user directly into Boundary, without any
+provisioned resources.
+
+Upon completion of the setup wizard you are logged into your HCP Boundary
+instance's Global scope via the web UI. This is the default scope for all new
+Boundary clusters.
+
+Click **Auth Methods**.
+
+Click the **copy icon** for the Password **Auth Method ID**. The ID is needed to
+authenticate to HCP Boundary using the CLI.
+
+<Tabs>
+<Tab heading="CLI">
+
+
+In the HCP Boundary portal, open the **Boundary Overview** page and click the
+**copy icon** in the **Copy this into Boundary Desktop** section. This copies
+your environment’s Boundary **Origin URL**.
+
+Open a terminal session and set the `BOUNDARY_ADDR` environment variable to the
+copied origin URL.
+
+```shell-session
+$ export BOUNDARY_ADDR=<boundary-origin-url>
+```
+
+Log in with the administrator credentials you created when you deployed the HCP
+Boundary instance.
+
+```shell-session
+$ boundary authenticate password \
+   -auth-method-id=COPIED_AUTH_ID \
+   -login-name=ADMIN_USERNAME \
+   -password=ADMIN_PASSWORD
+```
+
+You are now logged into your HCP Boundary instance's Global scope via the CLI.
+This is the default scope for all new Boundary clusters.
+
+If you are on Unix-like operating system (other than macOS/Darwin), you may get
+an error indicating that the token could not be stored, as the freedesktop.org
+Secret Service implementation is not always available. On these systems, you can
+work around this by installing `dbus-x11` and `gnome-keyring` using your package
+manager, then creating and unlocking the default keyring with the following,
+substituting in a password of your choice for "foobar" (but ending with `\n`).
+You can also avoid putting the password on the command line by running the
+`gnome-keyring-daemon` commands directly and entering in the password, followed
+by a newline (return) and an EOF (`Ctrl+D`).
+
+- `eval "$(printf 'foobar\n' | gnome-keyring-daemon --unlock)"`
+- `eval "$(printf 'foobar\n' | gnome-keyring-daemon --start)"`
+
+This would have to be run in each shell.
+
+If you're unable to install these packages, or don't want to, you can tell the
+Boundary authenticate command to not save the token to the operating system's
+key manager by setting `-token-name=none` flag or `BOUNDARY_TOKEN_NAME=none` env
+variable when running `boundary authenticate`. You'll be responsible for setting
+the token in subsequent commands via `-token` flag or `BOUNDARY_TOKEN` env
+variable. An easy way to do this would be to use the `-format=json` flag along
+with `jq` to pull the token value out of the response and place it wherever you
+wish, then create a command alias for `boundary` that sources that value into
+the environment or the `-token` flag.
+
+~> Token storage on \*nix systems has been more problematic than we expected.
```

Comment:
This issue is closed and we shouldn't encourage people to comment on it. In addition this ticket is outdated; it was created before we settled on `pass` as the default.

Reply:
thanks for the context! Let's sync on the alternative `pass` method.